README.md

```diff
@@ -2,7 +2,17 @@
 
 # Fair Data Protocol Storage
 
-> Typescript implementation of https://github.com/fairDataSociety/fairOS-dfs
+**FDP Storage** is a serverless web3 filesystem for organizing users' personal data implement in Typescript.
+
+Such data is stored using certain structures that allow the data created in one dApp to be interpreted in another dApp. The current implementation allows to create and manage pods (similar to disks in file systems), directories, and files.
+
+The library requires the [Bee](https://github.com/ethersphere/bee) endpoint of the node to interact with the data. If you plan to do write operations, you will need to specify [postage batch id](https://docs.ethswarm.org/docs/access-the-swarm/keep-your-data-alive). The library can work both with nodes connected to the mainnet network and with local nodes. To run a local test node, you can use [FDP Play](https://github.com/fairDataSociety/fdp-play).
```

Comment:
I would remove the network related part since it is possible to use any network with the library (e.g. testnet as well)
```suggestion
The library requires the API endpoint of a [Bee](https://github.com/ethersphere/bee) node to interact with the data. If you plan to do write operations, you will need to specify [postage batch id](https://docs.ethswarm.org/docs/access-the-swarm/keep-your-data-alive).
To run a local test node trying out the functionalities, you can use [FDP Play](https://github.com/fairDataSociety/fdp-play).
```